multires-lyrics-search/app.py

```diff
@@ -97,8 +97,8 @@ def dryrun():
               type=click.Choice(['index', 'query', 'index_restful', 'query_restful', 'dryrun'], case_sensitive=False))
 @click.option('--num_docs_query', '-n', default=100)
 @click.option('--num_docs_index', '-n', default=600)
-def main(task, num_docs_query, num_docs_index):
-    config(task)
+def main(task, num_docs_index):
```

Comment:
`num_docs_query` isnt used anymore now is it? If so then lets remove it from the parameters?